README.md

```diff
@@ -80,59 +80,78 @@ We have published a React Native NFC course with [newline.co](https://www.newlin
 The simplest (and most common) use case for this library is to read `NFC` tags containing `NDEF`, which can be achieved via the following codes:
 
 ```javascript
-import React from 'react';
-import {View, Text, TouchableOpacity, StyleSheet} from 'react-native';
-import NfcManager, {NfcTech} from 'react-native-nfc-manager';
-
-// Pre-step, call this before any NFC operations
-NfcManager.start();
-
-function App() {
-  async function readNdef() {
+import { StatusBar } from 'expo-status-bar';
```

Comment:
@ShiriNmi1520 it's awesome! I think we can remove this `expo-status-bar` stuff in case our user uses `react-native-cli`

Reply:
Sure I'll commit a change, hang on